### ISSUE TITLE
camera is missing in 0.5.0

### ISSUE THREAD

**Issue by SET001** [ggez/ggez-goodies#23]
it appears that camera module is being commented in 0.5.0:

`//pub mod camera;`

https://docs.rs/crate/ggez-goodies/0.5.0/source/src/lib.rs

**Comment by SET001** [comment]
I finally managed to make it work after directly selecting repo and branch in `.toml` file

```
ggez-goodies = { git="https://github.com/ggez/ggez-goodies", branch = "master"}
```

**Comment by icefoxen** [comment]
Yeah because it broke with the 0.4 -> 0.5 transition, and it didn't seem terribly useful in the first place.
